public/keymaps/treadstone32_default.json

```
@@ -0,0 +1,33 @@
+{
+    "keyboard": "treadstone32",
+    "keymap": "treadstone32_default",
+    "layout": "LAYOUT",
+    "layers": [
+        [
+            "KC_Q", "KC_W", "KC_E", "KC_R", "KC_T", "KC_Y", "KC_U", "KC_I", "KC_O", "KC_P",
+            "LSFT_T(KC_A)", "KC_S", "KC_D", "KC_F", "KC_G", "KC_H", "KC_J", "KC_K", "KC_L", "RSFT_T(KC_ENT)",
+            "LCTL_T(KC_Z)", "LALT_T(KC_X)", "LGUI_T(KC_C)", "KC_V", "KC_B", "KC_N", "KC_M", "KC_COMM", "KC_DOT", "RCTL_T(KC_SLSH)",
+            "LT(1,KC_BSPC)", "LT(2,KC_SPC)"
+        ],
+        [
+            "KC_F1", "KC_F2", "KC_F3", "KC_F4", "KC_F5", "KC_MINS", "KC_EQL", "KC_LBRC", "KC_RBRC", "KC_BSLS",
+            "LSFT_T(KC_F6)", "KC_F7", "KC_F8", "KC_F9", "KC_F10", "KC_NO", "KC_NO", "KC_NO", "KC_SCLN", "KC_QUOT",
+            "LSFT_T(KC_F11)", "LALT_T(KC_F12)", "KC_ESC", "KC_TAB", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_GRV",
+            "KC_TRNS", "MO(3)"
+        ],
+        [
+            "KC_1", "KC_2", "KC_3", "KC_4", "KC_5", "KC_6", "KC_7", "KC_8", "KC_9", "KC_0",
+            "KC_LSFT", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_LEFT", "KC_DOWN", "KC_UP", "KC_DOWN", "KC_RSFT",
+            "KC_LCTL", "KC_LALT", "KC_LGUI", "KC_NO", "KC_NO", "KC_MINS", "KC_RO", "KC_COMM", "KC_DOT", "RCTL_T(KC_SLSH)",
+            "KC_NO", "KC_TRNS"
```

I'm not 100% on what happens with layer change codes when you're doing two jumps, but to be safe, I think both of these should be `KC_TRNS`, for this layer and the next.

```suggestion
            "KC_TRNS", "KC_TRNS"
```

I specify this because I don't recall if doing `LT(1,KC_BSPC)` -> `MO(3)` and then releasing the `LT(1,KC_BSPC)` while still holding the `MO(3)` will actually keep the user on Layer 3.